website/sales/api/v2/admin/serializers/shift.py

```diff
@@ -36,13 +36,17 @@ class Meta:
             "end",
             "products",
             "total_revenue",
+            "total_revenue_paid",
             "num_orders",
             "product_sales",
         )
 
     total_revenue = serializers.DecimalField(
         max_digits=10, decimal_places=2, min_value=0, read_only=True
     )
+    total_revenue_paid = serializers.DecimalField(
```

Comment:
Consider using PaymentAmountSerializer (look for other uses if you need examples), also for total_revenue.